### PR TITLE
Clean Up Merchant List

### LINKED ISSUE #194: Addition to the Merchant page under Services - ELG
Please add to the Merchant page under Services;

Text: Emmanuel Galang, Canada Immigration and Refugee Lawyer
Link: www.galanglaw.com

Thank you!

### LINKED ISSUE #228: Please add a new site to the MONERO MERCHANTS AND SERVICES DIRECTORY
Please add algoStrategic.com to the MONERO MERCHANTS AND SERVICES DIRECTORY under the Services section.

algoStrategic - Internet Marketing and Web Development

### LINKED ISSUE #281: Payment Gateway Paybee Changed
Payb.ee is now live (no longer beta) and has rebranded as GloBee with a new url:  http://globee.com/

### LINKED ISSUE #285: Add Store to Merchant list
Hi, 

I own https://synntechgaming.com and would like to be added to the list of merchants accepting Monero. We build high-end custom desktops and offer professional services based out in NJ and LA. We've just announced that we will be accepting BTC, ETH and XMR as form of payment.

Thanks.

-Synntech

### LINKED ISSUE #326: Add SilverRound.com to merchants list.
We're a gold and silver dealer in California that began accepting Monero a few weeks ago. Feel free to contact me at charles@silverround.com if you have any questions.

https://silverround.com/

### LINKED ISSUE #360: Adding monero merchant: NiceVPS.net
Hi, would love to be added to the Monero merchants list. NiceVPS.net - VPS & Hosting (https://nicevps.net)
We already have a forum topic: https://forum.getmonero.org/3/merchants-and-marketplace/88159/nicevps-cheap-anonymous-monero-hosting-fast-protected-vps-starting-1-99-mo-1gbps

Thanks

### LINKED ISSUE #375: Please add MSvB HW Design to Services
Please add the following two lines of text to:

https://getmonero.org/community/merchants/

---- Begin text ----
MSvB Hardware Design
Using a secure element
---- End text ----

The hyperlink should navigate to:

"mailto:monerodev+sitesvcs@encambio.com?subject=Monero Services Hardware Design Request"

The code might resemble:

```
<div class="col-md-4 col-sm-6 col-xs-12">
    <a href="mailto:monerodev+sitesvcs@encambio.com?subject=Monero Services Hardware Design Request">MSvB Hardware Design,<br />Using a secure element</a>
</div>
```

Reasoning: Current efforts at creating custom Monero hardware designs such as a wallet or ATM machine. Requests are outstanding, so it would be good to have a place of contact on the site.

Thank you.

### LINKED ISSUE #390: XMR Lottery
Hi!  I am looking to get listed on the "Merchants & Services" page under entertainment.  I've built a lotto website here: https://www.xmrlotto.com.

Please let me know if there is anything I need to do!

Thanks!

### PR DESCRIPTION
**Pull request by QuickBASIC**
- alphabetizes merchants in each category 
- removes Bit.AC ([company has been disolved](https://beta.companieshouse.gov.uk/company/09986257))
- adds [Monero Integrations](http://monerointegrations.com/)
- Moved Elise Hawkins Nutritional Therapy to Services (per comment in #259. Issue is already closed.)
- closes #390 add XMRLotto.com
- closes #375 add MSvB HW Design to Services
- closes #281 (Globee is live (previously payb.ee) and at the URL https://globee.com instead of http://payb.ee.)
- closes #194 (add Galang Law Immigration and Refugee Services)
- closes #360 (adds NiceVPS.net; @mattcode55; I don't know if this PR will break #364 so I added here.)
- closes #228 (algostrategic.com added in #232 )
- closes #285 (synntechgaming.com added in #286 )
- closes #326 (SilverRound.com added in #328 )

### CODE REVIEW

**Comment by QuickBASIC** [comment]
@mattcode55 @MaxXor Do you mind reviewing this again. I had a merge conflict that I think I resolved correctly, but I don't know if I broke my PR. 😳

File: _data/merchants.yml [hunk 1]
```diff
@@ -2,10 +2,16 @@
   merchants:
     - name: "#monero-otc (OTC)"
       url: https://webchat.freenode.net?channels=%23monero-otc
+    - name: ArbitrCoin
+      url: https://arbitrcoin.com
+    - name: Alfacashier
+      url: https://www.alfacashier.com/
+    - name: Bitfinex (XMRUSD, XMRBTC)
+      url: https://www.bitfinex.com/
     - name: Bitsquare (decentralized)
```
Comment (MaxXor) [inline _data/merchants.yml]: Bitsquare is now Bisq (https://bisq.io/)

File: _data/merchants.yml [hunk 2]
```diff
@@ -55,130 +51,126 @@
...
-      url: https://flokinet.is
-    - name: Nerdzy Lawn Care
-      url: http://www.nerdzy.net/index.php/lawn-care/
-    - name: Njalla - privacy-aware domain registration
-      url: https://njal.la/
-    - name: Elise Hawkins Nutritional Therapy  
-      url: http://www.elisehawkinsnutritionaltherapy.com
-    - name: Koddos - hosting and DDoS protection services
-      url: http://koddos.net
+      url: xmr.to
```
Comment (MaxXor) [inline _data/merchants.yml]: Change to: https://xmr.to/

**Comment by QuickBASIC** [comment]
Fixed. Thank you.

I'm relatively new to using git/Github, so I wasn't sure that I had resolved the merge conflict correctly.

File: _data/merchants.yml [hunk 3]
```diff
@@ -175,4 +175,6 @@
     - name: MoneroDice
       url: https://monerodice.net
     - name: SafeDice
-      url: https://safedice.com
+      url: https://safedice.comet
```
Comment (MaxXor) [inline _data/merchants.yml]: `.comet` should be `.com` :smile:

Comment (QuickBASIC) [reply _data/merchants.yml]: Thanks. I blame lack of coffee and Hurricane Irma.

Comment (MaxXor) [reply _data/merchants.yml]: At least you didn't get hit by a comet.

**Comment by fluffypony** [comment]
Please rebase

**Comment by QuickBASIC** [comment]
Working on it.

**Pull request merged**